### DISCUSSION TITLE
Make it so you can't mount a ridable mob if they're aggro'd against you.

### DISCUSSION THREAD

**Discussion by ZepsiZola** [PurpurMC/Purpur#1267] [__Dismissed__] [answered]
I've been testing out ridable mobs and it seems like it should be pretty fun for SMP servers apart from the fact that you can get out of a tricky situation with an aggressive mob by just right-clicking it and sitting on it. I think there should be a way to make it so that you can only ride the mob when it's not aggressive towards you i.e. by sneaking up on it.

**Comment by RhythmicSys** [comment]
Sounds like a doable feature request for PurpurExtras

**Reply by ZepsiZola** [reply]
Is PurpurExtras a plugin?

**Reply by RhythmicSys** [reply, marked as answer]
Yeah, a plugin for features that would work better as a plugin than integrated into the software- in this case it would be really easy to write the case for canceling the mount if the mob is aggro'd instead of rewriting it inside the software itself
[PurpurExtras Repo](https://github.com/PurpurMC/PurpurExtras)
